src/npmPackage.js

```diff
@@ -46,6 +49,30 @@ export default function npmPackageFactory(rootDir = '', quiet = true) {
         stdio: quiet ? 'ignore' : 'inherit'
     });
 
+    /**
+     * Run any npm/npx command and wrap result in a Promise
+     * @param {String} command
+     * @returns {Promise} - resolves if command ran without errors, with data returned by command if any
+     */
+    const runCommand = (npmNpx, command, spawnOptions = {}) => {
+        return new Promise((resolve, reject) => {
+            if (typeof command !== 'string') {
+                return reject(new TypeError(`Invalid argument type: ${typeof command} for ${npmNpx}Command (should be string)`));
```

Comment:
If `command` can be non-string then `npmNpx` could be too? (Can we just combine both checks?)